tests/unit/Infrastructure/Manage/Dto/ManageEntityTest.php

```diff
@@ -117,4 +123,32 @@ public function test_is_requested_production_entity_copy()
         $this->assertTrue($entity->isRequestedProductionEntity(true));
         $this->assertFalse($entity->isRequestedProductionEntity(false));
     }
+
+    public function test_it_has_revision_notes()
+    {
+        $entity = ManageEntity::fromApiResponse(json_decode(file_get_contents(__DIR__ . '/fixture/saml20_sp_requested.json'), true));
+        $entity->setComments('comment on entity');
+        $this->assertSame('comment on entity', $entity->getRevisionNote());
+        $entity->setComments('another comment on entity');
+        $this->assertSame('another comment on entity', $entity->getRevisionNote());
+    }
+
+    public function test_it_has_entity_created_revision_notes_for_a_requested_state()
+    {
+        $entity = ManageEntity::fromApiResponse(json_decode(file_get_contents(__DIR__ . '/fixture/saml20_new_entity.json'), true));
+        $this->assertSame('Entity created', $entity->getRevisionNote());
+    }
+
+    public function test_it_has_entity_changed_revision_notes_for_a_requested_state()
+    {
+        $entity = ManageEntity::fromApiResponse(json_decode(file_get_contents(__DIR__ . '/fixture/saml20_changed_entity.json'), true));
+        $entity->setStatus(Constants::STATE_PUBLICATION_REQUESTED);
+        $this->assertSame('Entity changed', $entity->getRevisionNote());
+    }
+
+    public function test_it_has_changed_request_revision_notes_for_a_publisched_state()
+    {
+        $entity = ManageEntity::fromApiResponse(json_decode(file_get_contents(__DIR__ . '/fixture/saml20_changed_request.json'), true));
```

Comment:
The fixture name is very confusing. As the fixture results in a published production entity.